examples/objectServerExample/src/main/java/io/realm/examples/objectserver/model/CRDTCounter.java

```diff
@@ -13,38 +13,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package io.realm.examples.objectserver.model;
 
-import io.realm.RealmList;
+import io.realm.MutableRealmInteger;
 import io.realm.RealmObject;
 import io.realm.annotations.PrimaryKey;
+import io.realm.annotations.Required;
 
 /**
- * Counter class that is eventually consistent. Two devices can simultaneous increment this and eventually reach
- * the same value.
- *
- * @see <href ref="https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type">Conflict Free Replicated Data Structures</href>
+ * A named, conflict-free replicated data-type.
  */
 public class CRDTCounter extends RealmObject {
-
     @PrimaryKey
-    private long id;
-    private RealmList<CounterOperation> operations;
+    private String name;
```

Comment:
Since each user has her own Realm, this is strictly not needed as each user Realm would only have 1 object. It would also simplify the query which would just be `where(CRDTCounter.class).findFirstAsync()`

Reply:
Is that right?  Users don't share Realms in this app?

Reply:
I'm pretty sure they don't since you construct a new `SyncConfiguration` for each user.

Reply:
Yep.  That would do it, wouldn't it.